ksqldb-streams/src/main/java/io/confluent/ksql/execution/streams/SourceBuilder.java

```diff
@@ -192,9 +194,15 @@ private static Formats of(final KeyFormat keyFormat, final FormatInfo valueForma
       implements ValueTransformerWithKeySupplier<K, GenericRow, GenericRow> {
 
     private final int pseudoColumnVersion;
+    private final int numPseudoColumnsToMaterialize;
 
     AddPseudoColumnsToMaterialize(final int pseudoColumnVersion) {
       this.pseudoColumnVersion = pseudoColumnVersion;
+      this.numPseudoColumnsToMaterialize =
```

Comment:
The other `ValueTransformerWithKeySupplier` in this file also needs an analogous change.